src/dune_rules/enabled_if.ml

```diff
@@ -53,12 +55,12 @@ let decode_value ~allowed_vars ?(is_error = true) () =
     Blang.Ast.decode
       ~override_decode_bare_literal:None
       (String_with_vars.decode_manually (fun env var ->
+         let name = Dune_lang.Template.Pform.name var in
          match Dune_lang.Template.Pform.payload var with
```

Comment:
Why do we need this pattern match at all?

Reply:
Yes, there's no need,  I'll refactor.  I just kept it because it was there before.